lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,533 @@
+
+  process.env.CollectionsTable = cryptoRandomString({ length: 10 });
+  process.env.ExecutionsTable = cryptoRandomString({ length: 10 });
+  process.env.GranulesTable = cryptoRandomString({ length: 10 });
+
+  collectionsModel = new Collection();
+  await collectionsModel.createTable();
+
+  executionsModel = new Execution();
+  await executionsModel.createTable();
+
+  granulesModel = new Granule();
+  await granulesModel.createTable();
+
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.filePgModel = new FilePgModel();
+
+  // Store the CMR password
+  process.env.cmr_password_secret_name = cryptoRandomString({ length: 5 });
+  await secretsManager().createSecret({
+    Name: process.env.cmr_password_secret_name,
+    SecretString: cryptoRandomString({ length: 5 }),
+  }).promise();
+
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+});
+
+test.beforeEach(async (t) => {
+  const collectionPgModel = new CollectionPgModel();
+  const testCollection = fakeCollectionRecordFactory();
+
+  const collectionResponse = await collectionPgModel.create(
+    t.context.knex,
+    testCollection
+  );
+  t.context.testCollection = testCollection;
+  t.context.collectionCumulusId = collectionResponse[0];
+  t.context.collectionPgModel = collectionPgModel;
+
+  const executionPgModel = new ExecutionPgModel();
+  const testExecution = fakeExecutionRecordFactory();
+
+  const executionResponse = await executionPgModel.create(
+    t.context.knex,
+    testExecution
+  );
+  t.context.testExecution = testExecution;
+  t.context.executionCumulusId = executionResponse[0];
+  t.context.executionPgModel = executionPgModel;
+
+  t.context.testGranule = generateTestGranule(testCollection, testExecution.arn);
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.files).del();
+  await t.context.knex(tableNames.granules).del();
+  await t.context.knex(tableNames.collections).del();
+  await t.context.knex(tableNames.executions).del();
+});
+
+test.after.always(async (t) => {
+  await granulesModel.deleteTable();
+  await collectionsModel.deleteTable();
+  await executionsModel.deleteTable();
+
+  await s3Utils.recursivelyDeleteS3Bucket(bucket);
+
+  await secretsManager().deleteSecret({
+    SecretId: process.env.cmr_password_secret_name,
+    ForceDeleteWithoutRecovery: true,
+  }).promise();
+
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test.serial('migrateGranuleRecord correctly migrates granule record', async (t) => {
+  const {
+    collectionCumulusId,
+    executionCumulusId,
+    granulePgModel,
+    knex,
+    testGranule,
+  } = t.context;
+
+  await migrateGranuleRecord(testGranule, knex);
+
+  const record = await granulePgModel.get(knex, {
+    granule_id: testGranule.granuleId,
+    collection_cumulus_id: collectionCumulusId,
+  });
+
+  t.like(
```

Comment:
Are we using `like` here because of the extra camelCase dynamo fields? In other tests we use an explicit "omit list" which specifies those fields. That might be preferable just so it's super explicit and we don't get unexpected fields? Thoughts? 

See `collectionOmitList` in test-collections.js as an example.

Reply:
Hm yes, I opted to use `like` but perhaps it is better to be explicit and have an `omit list`.